participant/app/utils/config.server.ts

```diff
@@ -9,3 +9,5 @@ export const S3_PRESIGNED_URL_EXPIRATION =
 export const REGION = process.env.AWS_REGION || "us-west-2";
 export const ENDPOINT_URL = process.env.S3_ENDPOINT_URL || "";
 export const BUCKET = process.env.S3_BUCKET || "participant-uploads";
+export const MATOMO_URL_BASE =
+  process.env.MATOMO_URL_BASE || "https://dev-analytics.wic-services.org/"; // this is the analytics dashboard link
```

Comment:
Let's code this as an empty string as default.

And let's set matomo to use secure cookies for production images, but not for local development.
```suggestion
export const MATOMO_URL_BASE =
  process.env.MATOMO_URL_BASE || "";  // this is the analytics dashboard link
export const MATOMO_SECURE = (process.env.NODE_ENV == "production") ? true : false
```